spec/lib/dice_spec.rb

```diff
@@ -0,0 +1,22 @@
+require "rails_helper"
+
+RSpec.describe Dice do
+  describe "#roll_20_sided_dice" do
+    it "always rolls an integer between a 1 and a 20" do
+      100.times { expect(1..20).to include(Dice.roll_20_sided_dice) }
+    end
+  end
+
+  describe "#roll" do
+    it "formats the roll correctly" do
+      results = []
+      100.times { results << Dice.roll }
+
+      results.each do |result|
+        expect("1".."6").to include(result[0])
```

Comment:
We may want to make what we're testing here more explicit since the regular "roll" method is the foundation of everything.